model_demos/cv_demos/retinanet/pytorch_retinanet.py

```diff
@@ -0,0 +1,191 @@
+"""
```

Comment:
As this is our test, we don't need to reference their license. Only for copied/modified code we imported here.

Reply:
Removed Now!